tests/Loader/TestSuiteLoaderTest.php

```diff
@@ -12,11 +12,11 @@ class TestSuiteLoaderTest extends BaseTestCase
      */
     public function testLoadsSuite()
     {
-        $testSuite = $this->getMockBuilder('PHPUnit_Framework_TestSuite')
+        $testSuite = $this->getMockBuilder('PHPUnit\Framework\TestSuite')
             ->disableOriginalConstructor()
             ->getMock();
 
-        $configuration = $this->getMockBuilder('PHPUnit_Util_Configuration')
+        $configuration = $this->getMockBuilder('PHPUnit\Util\Configuration')
```

Comment:
I did not get to doing that everywhere. Step one would be to get the tests to work again :)